config/routes.rb

```diff
@@ -1,56 +1,15 @@
 Rails.application.routes.draw do
-  # The priority is based upon order of creation: first created -> highest priority.
-  # See how all your routes lay out with "rake routes".
+  root "home#index"
 
-  # You can have the root of your site routed with "root"
-  # root 'welcome#index'
+  resources :home, only: [:edit, :update]
```

Comment:
I'm taking by the edit and update routes not being namespaced under admin, that we are going to do inline editing on the page is you are logged in as an admin! I think that'll turn out really cool and make it really easy for the admin of the site be able to quickly update the content! :purple_heart:  :purple_heart:  :purple_heart: 

I did the rake routes to see what the difference would be between having each of the indexes broken out instead of just doing `resources :home, only: [:edit, :udpate, :index]` and it looks like the routes turn out the same but instead of calling home_index_path, you can now just call home_path. I wouldn't of thought of that and think it is more readable that way! :book: :+1: 

In case I am not making sense here is the screenshot of how the routes print as the PR is of now, and then if we add index up with the `only: [:edit, :udpate, :index]`
![screen shot 2015-04-20 at 8 51 33 pm](https://cloud.githubusercontent.com/assets/8313881/7244854/1aef103c-e79f-11e4-8a2d-ec36e64ce633.png)

I think it's worth having the extra line in the routes to be able to just call home_path, etc. Pretty cool! :+1: